zipkin-collector-service/build.gradle

```diff
@@ -1,6 +1,13 @@
 apply plugin: 'application'
 mainClassName = 'com.twitter.zipkin.collector.Main'
 
+run {
+    // When configKey is dev, collector and query share ../zipkin.db
+    workingDir rootProject.projectDir
```

Comment:
I suggest changing this to `rootProject.buildDir` that way the generated DB file will be placed in `build/` and naturally ignored by git and cleaned up by `./gradlew clean`